ramls/check-in.json

```
@@ -18,6 +18,20 @@
       "description": "Id of item that has checked in",
       "$ref": "raml-util/schemas/uuid.schema"
     },
+    "itemStatus": {
```

I think it might be worth making it explicit that this is the item status prior to check in.

```suggestion
    "itemStatusPriorToCheckIn": {
```